csp.py

```diff
@@ -517,8 +517,8 @@ def display(self, assignment):
 
 def flatten(seqs): return sum(seqs, [])
 
-easy1 = '..3.2.6..9..3.5..1..18.64....81.29..7.......8..67.82....26.95..8..2.3..9..5.1.3..'  # noqa
```

Comment:
I don't understand what these #noqa lines are for -- these lines are under 100 chars.